web/src/config.ts

```diff
@@ -15,21 +15,21 @@ export type Config = {
   logoUrl?: string;
   coverImageUrl?: string;
   cesiumIonAccessToken?: string;
-  webDomain: string;
+  editorUrl: string;
 };
 
 const env = import.meta.env;
 
 export const defaultConfig: Config = {
   api: env.REEARTH_CMS_API || "/api",
-  auth0Audience: env.REEARTH_CMS_AUTH0_AUDIENCE || "https://api.test.reearth.dev",
-  auth0Domain: env.REEARTH_CMS_AUTH0_DOMAIN || "reearth-oss-test.eu.auth0.com",
-  auth0ClientId: env.REEARTH_CMS_AUTH0_CLIENT_ID || "k6F1sgFikzVkkcW9Cpz7Ztvwq5cBRXlv",
+  auth0Audience: env.REEARTH_CMS_AUTH0_AUDIENCE,
+  auth0Domain: env.REEARTH_CMS_AUTH0_DOMAIN,
+  auth0ClientId: env.REEARTH_CMS_AUTH0_CLIENT_ID,
```

Comment:
Why did you remove the default values?

Reply:
I discussed about this with Inoue-san, and he said the default values should not be made public since the test environment is not open for general users.

Reply:
Okay, then you can remove it.